tensorflow/contrib/android/java/org/tensorflow/contrib/android/TensorFlowInferenceInterface.java

```diff
@@ -118,6 +192,7 @@ public TensorFlowInferenceInterface(InputStream is) {
    *
    * @param outputNames A list of output nodes which should be filled by the inference pass.
    */
+
```

Comment:
Can you make a pass and delete errant blank lines like this please? No blank lines are necessary after comments or after opening braces or before closing braces, and a single blank line is preferred after closing braces (depending on context).